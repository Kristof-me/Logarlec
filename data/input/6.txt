create room r1
create room r2
effect gas r1
effect wet r2
merge  r1 r2
status  r1 effects