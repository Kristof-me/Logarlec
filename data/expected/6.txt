gáz: 10
víz: 10